closed/custom/modules/java.base/Copy.gmk

```diff
@@ -30,6 +30,22 @@ $(call openj9_copy_files,, \
 	$(OPENJ9_VM_BUILD_DIR)/include/jvmti.h \
 	$(INCLUDE_TARGET_DIR)/jvmti.h)
 
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/j9cfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/j9cfg.h)
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/omr/omrcfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/omrcfg.h)
+else # OPENJ9_ENABLE_CMAKE
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/include/j9cfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/j9cfg.h)
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/omr/include_core/omrcfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/omrcfg.h)
+endif # OPENJ9_ENABLE_CMAKE
```

Comment:
Suggest extracting the differences so this is clearer:
```
ifeq (true,$(OPENJ9_ENABLE_CMAKE))
  CONFIG_HEADERS := j9cfg.h omr/omrcfg.h
else
  CONFIG_HEADERS := include/j9cfg.h omr/include_core/omrcfg.h
endif

$(foreach file, $(CONFIG_HEADERS), \
	$(call openj9_copy_files,, \
		$(OPENJ9_VM_BUILD_DIR)/$(file) \
		$(SUPPORT_OUTPUTDIR)/modules_include/java.base/$(notdir $(file))))
```

Reply:
Updated.